draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -996,12 +1048,20 @@ when presenting to a downstream party.
 In some scenarios this behavior could be desirable,
 but if it is not, Issuers need to support and Verifiers need to enforce Key Binding.
 
-## Integrity of Presentation
+## Malleability of SD-JWTs and SD-JWT-KBs
```

Comment:
I had to google "malleable", but guess that's only me..?

Reply:
Integrity is probably a more well known term

Reply:
During a call earlier today @danielfett said he'd suggest changes to rework the text in this section to avoid the use of "malleable" and rather talk about what integrity is and is not provided in each case.  

In the meantime, this would update the subsection title accordingly: 

```suggestion
## Integrity of SD-JWTs and SD-JWT-KBs
```

Reply:
I hesitate to make this change, because "Integrity" is about what can't be changed, and this section is about what *can* be changed.

Reply:
This and the proposed changes below say essentially the same thing. But just re-framed using "Integrity" and talking about what can't be changed, which also then describes what can be changed. I belive it's a more productive way to present the concepts. And it avoids the word "malleable" which @Sakurann (and the editors) didn't believe was particularly appropriate.

Reply:
trying to make this subsection title malleable again 
```suggestion
## Integrity of SD-JWTs and SD-JWT-KBs
```